### ISSUE TITLE
Subscribe form

### ISSUE THREAD

**Issue by markendley** [northbranding/TIAW#7]
Hey Joyce,

<img width="1094" alt="Screenshot 2024-08-06 at 12 13 26 pm" src="https://github.com/user-attachments/assets/ccfcb3d0-3c4e-4fa1-946e-660276a88ec1">

Option A: The name and email gets saved into the WP admin and the client gets an email notification.

Option B: The name/email gets sent to a third party of their choice.

I know I can do it quick and easy with...
- Mailchimp
- Active Campaign
- Campaign Monitor
- Klaviyo (rising in popularity for some reason, but it's slightly tougher for me to customize and make it work with the design)

If they have another CRM in mind I might need to check it to make sure I can integrate it with the website but most do.

**Comment by markendley** [comment]
(Allow me to turn it off until the client decides what they wanna do)

**Comment by northbranding** [comment]
Thanks Mark! Have queried the client on this, will get back to you as soon as we hear back.

**Comment by northbranding** [comment]
Hi Mark, it seems the client has requested more information on this.

**Comment by markendley** [comment]
Sure, what would they like to know?

If they're just unsure, I recommend Option A. Most CRMs like Mailchimp and Active Campaign offer a feature that makes it easy for me to import subscribers.

**Comment by northbranding** [comment]
Hi think they would like to know which would be easiest to manage, i do not think they have any CRM in place. 

What would happen with Option A? they get a notification, and then then can extract the email from the backend? is there a way that in the notification, they get told what the email address is and the details of the person?

**Comment by markendley** [comment]
Yes, after someone submits the form their Name and Email get saved into the WP database, the client can login, go to a page and see the list of all their subscribers.

Yes, somewhere on that page is a button so they can export the list as a csv file.

Yes I can turn on email notifications, so the client gets notified everytime they get a new subscriber.

Obviously the only information the client will get is the Name and Email (ie, whatever the form has asked for), but we can also record the time and date of when the form was submitted.

**Comment by northbranding** [comment]
Thanks Mark, the client will probably not want to have to log into the backend of the website to retrieve the name and email, as long as the info gets sent to their email of choice I think that would be best. Will relay what you've confirmed, and see what they say.

**Comment by northbranding** [comment]
Newsletter submission forms to go to [President@tiaw.org](mailto:President@tiaw.org)
